app/service/rest.py

```diff
@@ -312,6 +318,29 @@ def update_whitelist(service_id):
         return '', 204
 
 
+@service_blueprint.route('/<uuid:service_id>/deactivate', methods=['POST'])
+def deactivate_service(service_id):
+    service = dao_fetch_service_by_id(service_id)
+
```

Comment:
It would be good to wrap this in a transaction, so that a service can not get into a state where it is inactive but has active api keys or templates.